lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -167,65 +168,79 @@ class TestVertSplitTestVert(SplitTestBase):
         # We define problem compenents that we need but don't explicitly call elsewhere.
         # pylint: disable=unused-variable
-        super(TestVertSplitTestVert, self).setUp()
+        super(TestSplitTestVertVert, self).setUp()
 
-        # vert <- split_test
-        # split_test cond 0 = vert <- {video, problem}
-        # split_test cond 1 = vert <- {video, html}
-        vert1 = ItemFactory.create(
-            parent_location=self.sequential.location,
-            category="vertical",
-            display_name="Split test vertical",
-        )
         c0_url = self.course.id.make_usage_key("vertical", "split_test_cond0")
         c1_url = self.course.id.make_usage_key("vertical", "split_test_cond1")
 
         split_test = ItemFactory.create(
-            parent_location=vert1.location,
+            parent_location=self.sequential.location,
             category="split_test",
             display_name="Split test",
             user_partition_id='0',
             group_id_to_child={"0": c0_url, "1": c1_url},
         )
 
-        cond0vert = ItemFactory.create(
+        # vert <- split_test
+        # split_test cond 0 = vert <- {video, problem}
+        # split_test cond 1 = vert <- {video, html}
+        vert0 = ItemFactory.create(
             parent_location=split_test.location,
             category="vertical",
             display_name="Condition 0 vertical",
             location=c0_url,
         )
-        video0 = self._video(cond0vert, 0)
-        problem0 = self._problem(cond0vert, 0)
 
-        cond1vert = ItemFactory.create(
+        vert1 = ItemFactory.create(
             parent_location=split_test.location,
             category="vertical",
             display_name="Condition 1 vertical",
             location=c1_url,
         )
+
+        cond0vert = ItemFactory.create(
+            parent_location=vert0.location,
+            category="vertical",
+            display_name="Condition 0 vertical child",
+        )
+        video0 = self._video(cond0vert, 0)
+        problem0 = self._problem(cond0vert, 0)
+
+        cond1vert = ItemFactory.create(
+            parent_location=vert1.location,
+            category="vertical",
+            display_name="Condition 1 vertical child",
+        )
         video1 = self._video(cond1vert, 1)
         html1 = self._html(cond1vert, 1)
 
+        self.included_usage_keys = [
+            [video0.scope_ids.usage_id, problem0.scope_ids.usage_id],
+            [video1.scope_ids.usage_id, html1.scope_ids.usage_id],
+        ]
+
+        self.excluded_usage_keys = [
+            [video1.scope_ids.usage_id, html1.scope_ids.usage_id],
+            [video0.scope_ids.usage_id, problem0.scope_ids.usage_id],
+        ]
 
-class TestSplitTestVert(SplitTestBase):
+
+class TestVertSplitTestVert(SplitTestBase):
     """
-    Tests related to xmodule/split_test_module
+    Tests a unit whose top-level vertical is constant and contains a split test determining content within that
```

Comment:
To avoid confusion with other existing terminology, use the word `sequential` instead of `unit` here.  The reason being that `unit` is equivalent to `vertical` in our nomenclature.